lib/containers/common.pm

```diff
@@ -53,7 +54,7 @@ sub install_podman_when_needed {
 }
 
 sub install_docker_when_needed {
-    my $host_os = shift;
+    my ($host_os, $allow_build) = @_;
```

Comment:
```suggestion
    my ($host_os, $is_sle) = @_;
```

Reply:
done. but i use `is_sle_host` to avoid confusion with the `is_sle` from the version_utils